demo/app/public/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <meta name="theme-color" content="#ffffff" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Demo Config Tool | Ad.js by Econify</title>
+  </head>
+
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <header>
+      <h1>Ads.js Demo Config Tool</h1>
+    </header>
+    <div class="container">
+      <div class="demo">
+        <div id="ad-slot-1"></div>
+      </div>
+      <div id="root"></div>
+    </div>
+    <footer>
+      <hr />
+      <img src="http://www.econify.com/assets/econify_logo.png" />
+      <small>Copyright &copy; 2018 Econify, LLC. All Rights Reserved.</small>
```

Comment:
I dont know our situation with licenses but this is old. Maybe just throw a `new Date().getFullYear()` in there or something

Reply:
This is updated now to dynamic year.